src/arena/game_state.rs

```diff
@@ -479,13 +523,23 @@ impl GameState {
         self.round_data = round_data;
     }
 
+    fn increment_player_acted_count(&mut self) {
+        let idx = self.round_data.to_act_idx;
+
+        if let Some(acted_count) = self.times_acted.get_mut(idx) {
+            *acted_count += 1;
+        }
+    }
+
```

Comment:
This won't be needed, when using https://github.com/elliottneilclark/rs-poker/blob/353e7c0cdb2b919a59bb8404ff059be8f29603f6/src/arena/historian/stats_tracking.rs#L75